src/main/java/seedu/address/logic/commands/sort/SortKey.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.logic.commands.sort;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+/**
+ * Sort Keys that can be used that can be used for sort keys.
+ */
+public enum SortKey {
```

Comment:
The header comment for SortKey may sound a little confusing 😅

Reply:
You are right! Will update this!